sdk/spring/spring-cloud-azure-testcontainers/pom.xml

```diff
@@ -83,6 +87,7 @@
       <groupId>org.testcontainers</groupId>
       <artifactId>azure</artifactId>
       <version>1.19.8</version> <!-- {x-version-update;springboot3_org.testcontainers:azure;external_dependency} -->
+      <optional>true</optional>
```

Comment:
If we make this one as optional, should users add this dependency explictly?

Reply:
the testcontainers azure dependency only adds support for CosmosDB Emulator. Yes, dependency should be declared explicitly in this case.